Sources/KingfisherManager+Rx.swift

```diff
@@ -12,14 +12,14 @@ import Kingfisher
 extension Reactive where Base == KingfisherManager {
     public func retrieveImage(with resource: Resource,
                               options: KingfisherOptionsInfo? = nil) -> Single<Image> {
-        return Single<Image>.create { [base] single in
+        return Single.create { [base] single in
```

Comment:
Similarly to https://github.com/RxSwiftCommunity/RxKingfisher/pull/11/files#r265926029
```suggestion
        return .create { [base] single in
```